docs/companion.md

```diff
@@ -643,10 +635,12 @@ risk.**
 
 :::
 
-#### `corsOrigins` `COMPANION_CLIENT_ORIGINS`
+#### `corsOrigins` `COMPANION_CLIENT_ORIGINS` (required)
 
-Allowed CORS Origins (default `true`). Passed as the `origin` option in
-[cors](https://github.com/expressjs/cors#configuration-options))
+Allowed CORS Origins. Passed as the `origin` option in
```

Comment:
Can we explain what should be passed? Entire URI as string? Compatible with everything `cors` supports?

Reply:
There's a link to the docs (also it's a bit unrelated to this PR)

Reply:
That's what makes it confusing, I read that link I and I pass a regex. Then it crashes because `postMessage` does not support a regex. So it is related to this PR to clarify what is now support and remove that link

Reply:
No no, a regex would work just fine